docs/broadcast-guide.md

```diff
@@ -0,0 +1,89 @@
+# Guide to using Broadcast Variables
+
+This is a guide to show how to use Broadcast variables in .NET for Apache Spark.
```

Comment:
nit: Should we capitalize broadcast in the middle of sentences?

Reply:
Good point, I will change that